create_db.py

```diff
@@ -0,0 +1,53 @@
+from src.game_database import create_database
+from src.clean_data import read_data,filter_data,save_files
+from config import config
+
+from tqdm import tqdm
+
+import logging
```

Comment:
might want to create an actual logger object, rather than using logging lib directly. logger objects are more configurable.

Reply:
Definitely. Sorry for not including this initially!